lab_3_ann_retriever/main.py

```diff
@@ -68,11 +54,10 @@ def load_vector(state: dict) -> Vector | None:
     Load vector from state.
 
     Args:
-        state (dict): State of the vector to load from
+        state (dict): state of the vector to load from.
```

Comment:
i wont write about this. Revert all changes like that

Reply:
fixed